### PR TITLE
MGMT-3574 Add e2e-metal-assisted-single-node-periodic test to assisted-test-infra

### PR DESCRIPTION
**Pull request by YuviGold**

### CODE REVIEW

File: ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml [hunk 1]
```diff
@@ -65,6 +65,15 @@ tests:
   steps:
     cluster_profile: packet
     workflow: baremetalds-assisted
+- as: e2e-metal-assisted-single-node-periodic
```
Comment (omertuc) [inline ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:68]: This looks like it's testing BiP specifically, maybe change the name to reflect that?
Or do we not care about non-bip SNO in assisted?

**Comment by romfreiman** [comment]
we don't care about non-bip in assisted flow

File: ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml (hunk 1 above)
Comment (tsorya) [reply ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:68]: assisted-single-node sounds like clear one

Comment (romfreiman) [reply ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:68]: in this way it's aligned with other jobs

**Pull request merged**